README.md

```diff
@@ -16,7 +16,34 @@ http://developers.google.com/api-client-library/php
 
 ## Installation ##
 
-For the latest installation and setup instructions, see [the documentation](https://developers.google.com/api-client-library/php/start/installation).
+The [Releases](https://github.com/google/google-api-php-client/releases) page lists all stable versions. Download any file
+with the name `google-api-php-client-[RELEASE_NAME].zip` for a package including this library and its dependencies.
+
+### Composer
+
+Install this library via [composer](https://getcomposer.org) by running the following command:
+
+```sh
+# change directories to the root of your project
+cd /path/to/my-project
+# if you don't already have composer, install it
+curl -sS https://getcomposer.org/installer | php
```

Comment:
Yes, but composer should still be encouraged, hence the example.
On Tue, Nov 3, 2015 at 5:30 PM Takashi Matsuo <notifications@github.com>
wrote:

> In README.md
> <https://github.com/google/google-api-php-client/pull/729#discussion_r43832430>
> :
>
> > @@ -16,7 +16,34 @@ http://developers.google.com/api-client-library/php
> >
> >  ## Installation ##
> >
> > -For the latest installation and setup instructions, see [the documentation](https://developers.google.com/api-client-library/php/start/installation).
> > +The [Releases](https://github.com/google/google-api-php-client/releases) page lists all stable versions. Download any file
> > +with the name `google-api-php-client-[RELEASE_NAME].zip` for a package including this library and its dependencies.
> > +
> > +### Composer
> > +
> > +Install this library via [composer](https://getcomposer.org) by running the following command:
> > +
> > +```sh
> > +# change directories to the root of your project
> > +cd /path/to/my-project
> > +# if you don't already have composer, install it
> > +curl -sS https://getcomposer.org/installer | php
>
> Well, people who don't know composer can still download the zip file,
> right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/google-api-php-client/pull/729/files#r43832430>
> .
>